### PR TITLE
Addons update version

### PR DESCRIPTION
**Pull request by hellupline**
update addons version to keep up-to-date with [k8s addons
(https://github.com/kubernetes/kubernetes/tree/master/cluster/addons)

k8s-dashboard update improved UI, allow visualization of Storage, Service, Config

### CODE REVIEW

**Comment by hellupline** [comment]
I signed it!

**Comment by justinsb** [comment]
I'm a bit wary of merging the beta tags, but as soon as the release goes live, this looks great - thanks :-)

**Comment by yissachar** [comment]
Heapster and Dashboard have now released new non-beta versions.

**Comment by chrislovecnm** [comment]
we need a rebase on this.

File: addons/dashboard/v1.1.0.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ metadata:
   labels:
     app: kubernetes-dashboard
     k8s-addon: kubernetes-dashboard.addons.k8s.io
-    version: v1.1.0
+    version: v1.4.0-beta1
```
Comment (chrislovecnm) [inline addons/dashboard/v1.1.0.yaml]: What is the proper version now?

File: addons/dashboard/v1.1.0.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ spec:
     spec:
       containers:
       - name: kubernetes-dashboard
-        image: gcr.io/google_containers/kubernetes-dashboard-amd64:v1.1.0
+        image: gcr.io/google_containers/kubernetes-dashboard-amd64:v1.4.0-beta1
```
Comment (chrislovecnm) [inline addons/dashboard/v1.1.0.yaml]: Please update version

**Comment by chrislovecnm** [comment]
@hellupline you need to sign the new cla as well.  Kubernetes project moved to linuxfoudation cla.

**Comment by chrislovecnm** [comment]
I believe we have done the updates. I would like to close this out.

File: addons/dashboard/v1.1.0.yaml (hunk 1 above)
Comment (ajohnstone) [inline addons/dashboard/v1.1.0.yaml]: v1.4.1

**Comment by kris-nova** [comment]
Closing in favor of https://github.com/kubernetes/kops/pull/716

**Pull request closed without merging**